src/earn/EarnPoolInfoScreen.test.tsx

```diff
@@ -7,29 +7,75 @@ import EarnPoolInfoScreen from 'src/earn/EarnPoolInfoScreen'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { EarnPosition } from 'src/positions/types'
+import { getFeatureGate, getMultichainFeatures } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import { NetworkId } from 'src/transactions/types'
 import { navigateToURI } from 'src/utils/linking'
 import MockedNavigator from 'test/MockedNavigator'
-import { createMockStore } from 'test/utils'
-import { mockArbUsdcTokenId, mockEarnPositions, mockTokenBalances } from 'test/values'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+import {
+  mockArbEthTokenId,
+  mockArbUsdcTokenId,
+  mockCusdTokenId,
+  mockEarnPositions,
+  mockTokenBalances,
+} from 'test/values'
+
+jest.mock('src/statsig', () => ({
+  getMultichainFeatures: jest.fn(),
+  getFeatureGate: jest.fn(),
+}))
 
 const mockPoolTokenId = mockEarnPositions[0].dataProps.depositTokenId
 
-const store = createMockStore({
-  tokens: {
-    tokenBalances: { [mockPoolTokenId]: mockTokenBalances[mockArbUsdcTokenId] },
-  },
-})
+function getStore({
+  balance = '0',
+  includeSameChainToken = false,
+  includeOtherChainToken = false,
+}: {
+  balance?: string
+  includeSameChainToken?: boolean
+  includeOtherChainToken?: boolean
+}) {
```

Comment:
```suggestion
} = {}) {
```
so you can just do `getStore()` when you don't need overrides